models/ParkingSpotRequest.ts

```diff
@@ -6,6 +6,9 @@ const ParkingSpotRequestSchema = new Schema(
     user: {
       type: Schema.ObjectId,
     },
+    schoolYear: {
```

Comment:
Unless the number is only for displaying, I think this could be better represented as a single number  or a pair of numbers.